example/dist/sprites-manifest.json

```diff
@@ -5,7 +5,7 @@
     "src/svgs/video.svg"
   ],
   "news": [
-    "src/svgs/smiley-surprised.svg",
-    "src/svgs/popcorn.svg"
+    "src/svgs/popcorn.svg",
+    "src/svgs/smiley-surprised.svg"
```

Comment:
@alexander-akait In [news.js#L1-L2](https://github.com/yoriiis/svg-chunk-webpack-plugin/blob/7e4571b2108abda1bbafb58510b692f707bb1c43/example/src/js/news.js#L1-L2), the order is `smiley-surprised` then `popcorn`.

I thought the order had to respect that of the declarations but now it's sorted alphabetically

Reply:
Do you need to have order by in source code? I just don't have it, but yeah, we have `debugId` and other ids to prevent it

Reply:
I don't think so, I'll leave as you did